qa/covid19_qa/elasticsearch_qa.py

```diff
@@ -0,0 +1,49 @@
+from elasticsearch import Elasticsearch
+from typing import Iterator
+
+from covid19_qa.pipeline import Instance
+
+# FIXME: The elasticsearch configuration should be a setting
+es = Elasticsearch([{'host': 'elasticsearch-covid', 'port': 9200}])
+#es = Elasticsearch([{'host': '0.0.0.0', 'port': 19200}])
```

Comment:
This can be the same always, right? Because it's inside a docker container (the host and local port will always be the same).

Reply:
It depends if we host both containers in the same machine or not. If we are using different machines without any orchestator (k8s) you need to manually expose the elastic search service and the qa container access directly to it.

Reply:
Oh, yeah, I was misunderstanding this :+1: